modules/apps/portal-cache/portal-cache-ehcache-impl/src/main/java/com/liferay/portal/cache/ehcache/internal/BaseEhcachePortalCacheManager.java

```diff
@@ -100,7 +100,15 @@ public void destroy() {
 
 	@Override
 	public PortalCache<K, V> fetchPortalCache(String portalCacheName) {
-		return _portalCaches.get(portalCacheName);
+		String cacheName = portalCacheName;
+
+		int indexOf = portalCacheName.indexOf(_SHARDED_SEPARATOR_);
+
+		if (indexOf != -1) {
+			cacheName = portalCacheName.substring(0, indexOf);
+		}
+
+		return _portalCaches.get(cacheName);
 	}
```

Comment:
@marianoalvarosaiz, please do SF. 
1. The var cacheName is unnecessary. You may directly use portalCacheName = portalCacheName.substring()
2. Please rename `indexOf ` for index. In our codes, for the kind of case, we named it as `index`.